ehrql/tables/beta/core.py

```diff
@@ -82,12 +83,57 @@ def age_on(self, date):
 
 
 @table
-class ons_deaths(EventFrame):
+class ons_deaths_raw(EventFrame):
     date = Series(
         datetime.date,
         description=(
             "Patient's date of death. "
             "Only deaths registered from February 2019 are recorded."
+            "This table may contain multiple entries per patient."
+        ),
+    )
+    place = Series(
+        str,
+        constraints=[
+            Constraint.Categorical(
+                [
+                    "Care Home",
+                    "Elsewhere",
+                    "Home",
+                    "Hospice",
+                    "Hospital",
+                    "Other communal establishment",
+                ]
+            ),
+        ],
+    )
+    underlying_cause_of_death = Series(ICD10Code)
+    # TODO: Revisit this when we have support for multi-valued fields
+    cause_of_death_01 = Series(ICD10Code)
+    cause_of_death_02 = Series(ICD10Code)
+    cause_of_death_03 = Series(ICD10Code)
+    cause_of_death_04 = Series(ICD10Code)
+    cause_of_death_05 = Series(ICD10Code)
+    cause_of_death_06 = Series(ICD10Code)
+    cause_of_death_07 = Series(ICD10Code)
+    cause_of_death_08 = Series(ICD10Code)
+    cause_of_death_09 = Series(ICD10Code)
+    cause_of_death_10 = Series(ICD10Code)
+    cause_of_death_11 = Series(ICD10Code)
+    cause_of_death_12 = Series(ICD10Code)
+    cause_of_death_13 = Series(ICD10Code)
+    cause_of_death_14 = Series(ICD10Code)
+    cause_of_death_15 = Series(ICD10Code)
+
+
+@table
+class ons_deaths(PatientFrame):
+    date = Series(
+        datetime.date,
+        description=(
+            "Patient's date of death. "
+            "Only deaths registered from February 2019 are recorded."
+            "If a patient has multiple entries for death, this table only contains the first entry."
```

Comment:
As previously, this information applies to the table, so it should be added to the table description. Setting this aside, I think either more or less description is required. For example:

> In [the associated database table][1], a small number of patients have multiple registered deaths. This table contains the first registered death. The `ons_deaths_raw` table contains all registered deaths.

[1]: https://reports.opensafely.org/reports/opensafely-tpp-database-schema/#ONS_Deaths